plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/TntCommand.java

```diff
@@ -31,8 +32,8 @@ public void execute(CommandSender sender) {
                 ? now.isAfter(start) || now.isBefore(end)
                 : now.isAfter(start) && now.isBefore(end);
 
-        message = StringUtils.replace(message, "{PROTECTION_START}", config.tntProtection.time.startTime_);
-        message = StringUtils.replace(message, "{PROTECTION_END}", config.tntProtection.time.endTime_);
+        message = StringUtils.replace(message, "{PROTECTION_START}", config.tntProtection.time.startTime.format(LocalTimeTransformer.TIME_FORMATTER));
+        message = StringUtils.replace(message, "{PROTECTION_END}", config.tntProtection.time.endTime.format(LocalTimeTransformer.TIME_FORMATTER));
```

Comment:
Trochę głupie, że musimy to ręcznie wywoływać, powinno zwracać już gotową wartość tak jak wcześniej